### ISSUE TITLE
Post-publish "are you sure" message makes me anxious about whether my changes are save or not

### ISSUE THREAD

**Issue by lancewillett** [WordPress/gutenberg#3319]
Gutenberg version: 1.6.1
WP version: trunk

## Steps to Reproduce (for bugs)
1. Create a new post
2. Enter a title and some body text (paragraph block, for example)
3. Click the publish button once, then again to confirm

Tested with latest Chrome and Safari 11.0.1 on OS X 10.12.6.

## Expected Behavior
I'd expect to not see a warning about unsaved changes after publishing the post. When I next click to close the tab or go to a different link from within the tab I expect it to work correctly.

## Current Behavior
When I published the post, and tried to click to close the tab — I was blocked by the infamous AYS alert message like we have in [Calypso a lot](https://github.com/Automattic/wp-calypso/issues/9833)! (CC @aduth) — “Are you sure you want to leave this page?”

## Screenshot / Video

<img width="1281" alt="screen shot 2017-11-02 at 12 40 57" src="https://user-images.githubusercontent.com/66797/32346634-2995da92-bfcb-11e7-9b64-b14e04637d65.png">

<img width="532" alt="ays-gutenberg" src="https://user-images.githubusercontent.com/66797/32346136-9a904824-bfc9-11e7-879f-ad3144332f8b.png">

![gutenberg-ays](https://user-images.githubusercontent.com/66797/32346301-2c203b6e-bfca-11e7-8e01-37c1e1267e85.gif)

## Related Issues and/or PRs
https://github.com/WordPress/gutenberg/issues/3279

**Comment by aduth** [comment]
Thanks for the report. Duplicate of #3296

**Comment by lancewillett** [comment]
Gah! Sorry I missed the other one.

I did make a PR for something else. :)

https://github.com/WordPress/gutenberg/pull/3320
